t/27-consoles-vmware.t

```diff
@@ -232,15 +232,15 @@ subtest 'turning WebSocket into normal socket via dewebsockify' => sub {
     $vmware->_cleanup_previous_dewebsockify_process;
 
     # test error handling of dewebsockify
-    my $dewebsockify_cmd_start = "$bmwqemu::topdir/script/dewebsockify --listenport $tcp_port --websocketurl";
+    my $dewebsockify_cmd_start = "$bmwqemu::topdir/script/dewebsockify.pm --listenport $tcp_port --websocketurl";
```

Comment:
don't call a module like a script. Either import the module and call functions within or call the original script